modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -1867,12 +1893,28 @@ private void resizeCellSize(T cell) {
         }
     }
 
-    private List<T> getCells() {
+    /**
+     * Returns the list of cells displayed in the current viewport.
+     * <p>
+     * The cells are ordered such that the first cell in this list is the first in the view, and the last cell is the
+     * last in the view. When pixel scrolling, the list is simply shifted and items drop off the beginning or the end,
+     * depending on the order of scrolling.
+     *
+     * @return the cells displayed in the current viewport
+     * @since 12
+     */
+    protected List<T> getCells() {
         return cells;
     }
 
-    // Returns last visible cell whose bounds are entirely within the viewport
-    T getLastVisibleCellWithinViewPort() {
+    /**
+     * Returns the last visible cell whose bounds are entirely within the viewport. When manually inserting some rows,
+     * one may need to know which index are actually visible to the end user in the viewport.
+     *
```

Comment:
* "inserting some rows" remove "some"
* "which index are actually visible to the end user in the viewport" -> "which cell indices are visible in the viewport"